Fix camera buttons

skks1212 opened the pull request:
closes #2683

nihal467 commented:
![Screenshot 2022-06-12 161844](https://user-images.githubusercontent.com/57055998/173229585-807eb2da-c02a-4fcc-b9cd-8df3a03bac40.jpg)

@skks1212 @gigincg test failed, to see the full feed-in laptop screen, we need to scroll it, and make it fit the screen size

skks1212 commented:
@nihal467 I have changed the camera feed UI completely, and also added space for the 5 para monitor readings, which will be added in the future. I have removed the aspect ratio class so that the flex could work properly, but it will be of no problem as the video will just show black bars if out of ratio.
